src/core/core.helpers.js

```diff
@@ -723,7 +723,7 @@ module.exports = function(Chart) {
 	};
 	helpers.addEvent = function(node, eventType, method) {
 		if (node.addEventListener) {
-			node.addEventListener(eventType, method);
+			node.addEventListener(eventType, method, {passive: true});
```

Comment:
@asimkt while re-organizing our helpers, I moved `addEvent` and `removeEvent` in `platform.core.js` but also implemented this feature detection, I hope you don't mind (see #4424).